ui/app/src/components/UserManagement/utils.ts

```diff
@@ -30,7 +30,11 @@ export const minLengthMap = {
 };
 
 export const isInvalidEmail = (email: string) => {
-  const emailRegex = /^([\w\d._\-#])+@([\w\d._\-#]+[.][\w\d._\-#]+)+$/g;
+  // From https://emailregex.com/, according to the RFC 5322 standard
+  const emailRegex =
+    // Linter shows a couple of warnings, but the regex is correct
+    // eslint-disable-next-line
+    /^((?:[A-Za-z0-9!#$%&'*+\-\/=?^_`{|}~]|(?<=^|\.)"|"(?=$|\.|@)|(?<=".*)[ .](?=.*")|(?<!\.)\.){1,64})(@)((?:[A-Za-z0-9.\-])*(?:[A-Za-z0-9])\.(?:[A-Za-z0-9]){2,})$/gm;
```

Comment:
@sumerjabri @jmendeza @russdanner FYI please review expression. 
@sumerjabri @russdanner Not sure what's the best way of knowing this is the right thing to use?

@jvega190 what's the exact link? The expression I see on the emailregex.com home page for JS is not this one.

Reply:
@rart sorry, there was a change I didn't push. Actually I sent the PR with an incorrect branch. I'll close this and create the correct one. I'll re-add the comments.